scripts/bootstrap.py

```diff
@@ -104,6 +105,7 @@ def populate_known_hosts(hosts, dest_file):
 
 def main():
     try:
+        jenkins_agent_role = os.environ['JENKINS_AGENT_ROLE']
```

Comment:
Do we want this to be required or should this be moved to the optional section below?